Framework/API/inc/MantidAPI/ITableWorkspace.h

```diff
@@ -266,20 +266,20 @@ class MANTID_API_DLL ITableWorkspace : public API::Workspace {
   TableRowHelper getFirstRow() { return TableRowHelper(this, 0); }
 
   /// find method to get the index of integer cell value in a table workspace
-  virtual void find(size_t value, size_t &row, const size_t &col) = 0;
+  virtual void find(size_t value, size_t &row, size_t col) = 0;
```

Comment:
Just wanted to ask, do you know if its preferred in Mantid to pass in a `size_t` parameters without `const`? I thought that passing in as `const` was preferred, but I might be wrong!

Reply:
Cppcheck complains if `const` is missing when passing a parameter by reference and that parameter is not modified by the function.

Here, passing a `size_t` by const reference does not make sens. Because `size_t col` is not modified, it should be passed by value. I can still add the `const` but i do not think this is adding anything here.